src/lib/marketplaces/marketplaces.ts

```diff
@@ -3,6 +3,7 @@ import magicEden from "./magicEden";
 import digitalEyes from "./digitalEyes";
 import solanart from "./solanart";
 import alphaArt from "./alphaArt";
+import solSea from "./solsea";
```

Comment:
This import here is correct but Solsea.ts filename didn't match the case here. 
I believe that's what caused `yarn build` to fail.

Reply:
👍  will fix that